### ISSUE TITLE
Cannot add ultrasonic to api

### ISSUE THREAD

**Issue by xunxis966** [lgsvl/simulator#1073]
I try to add ultrasonic to vehicle. I can use with Simulation ( BorregasAve, no bridge, data collection ), but i can not use with API . The error is " Exception: Unknown data type UltrasonicData used for publisher in CyberRT bridge " . Im not sure what i do wrong, thank you.

**Comment by EricBoiseLGSVL** [comment]
@xunxis966 Are you using the ultrasonic for the CyberRT bridge?  06 release only supports Cyber.  Our next release will support ROS and ROS2

**Comment by xunxis966** [comment]
No, i want use with only api to show distance when loading scene. I'm using version 06.
